### PR TITLE
issue-2: Install storybook/addons, storybook/theming packages...

### PR DESCRIPTION
**Pull request by cfbauer**
Add CivicActions basic theming

### CODE REVIEW

File: .storybook/CivicActionsTheme.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { create } from '@storybook/theming';
+import logo  from './img/CA-Full-Logo-Red.svg';
+
+let red = '#D83933';
```
Comment (alex9000) [inline .storybook/CivicActionsTheme.js:4]: @cfbauer shouldn't these be const?

Comment (cfbauer) [reply .storybook/CivicActionsTheme.js:4]: Yes!

**Comment by cfbauer** [comment]
Wasn't sure if we needed the /addons package since there are already addons in the code but the documentation suggests installing it: https://storybook.js.org/docs/react/configure/theming

**Comment by erichuffman** [comment]
Looks like we want to keep all the `@storybook` packages under `devDependencies`. Can you reinstall with the `--save-dev` flag? Also, did you need to make updates/additions to the `main.js` (per slack conversations?)

Either way, things looked good locally to me, but note that the color of the button for collapsible sections in the left-side menu doesn't look like it [passes contrast checks](https://webaim.org/resources/contrastchecker/?fcolor=999999&bcolor=F5F5F0). Did we have a lighter branding color we can use?
image

**Comment by cfbauer** [comment]
From meeting: putting the contrast issue on hold for now since the USWDS site also shows the issue and it could be a larger task.

**Comment by erichuffman** [comment]
@cfbauer things look good, but I think you want to make sure the `package-lock.json` file is included in your PR since it should show diffs too. Thx!

**Comment by cfbauer** [comment]
Not sure why `package-lock.json` is so different after running `npm install` but here we are.

**Review by erichuffman — APPROVED** [review]

**Pull request merged**